packages/emotion-utils/src/index.js

```diff
@@ -7,6 +7,13 @@ export function memoize(fn) {
   }
 }
 
+export const STYLES_KEY = '__emotion_styles'
+export const TARGET_KEY = '__emotion_target'
+
+export function isEmotionElement(fn) {
```

Comment:
Could you please inline this function. I'd rather not expose it, especially in a place where people can import it since it means people could start relying on it. I'm okay with exporting the keys since I think it's clear that they're implementation details.